build.gradle.kts

```diff
@@ -79,7 +79,23 @@ apiValidation {
    * Classes (fully qualified) that are excluded from public API dumps even if they
    * contain public API.
    */
-//  ignoredClasses.add("com.company.BuildConfig")
+  ignoredClasses.add("com.mapbox.maps.extension.androidauto.BuildConfig")
+  ignoredClasses.add("com.mapbox.maps.extension.style.BuildConfig")
+  ignoredClasses.add("com.mapbox.maps.extension.localization.BuildConfig")
+  ignoredClasses.add("com.mapbox.maps.module.telemetry.BuildConfig")
+  ignoredClasses.add("com.mapbox.maps.plugin.camera.animation.BuildConfig")
```

Comment:
noticed the package name in the plugin-animation's AndroidManifest file does not match the package of the source code `com.mapbox.maps.plugin.animation`, but I assume it should be ok.

Reply:
We should fix this in v11